bokeh/models/glyphs.py

```diff
@@ -346,6 +346,11 @@ def __init__(self, **kwargs):
 
     """)
 
+    level = Enum('image', 'underlay', 'glyph', default='underlay', help="""
+    Specifies the level in which to render the image.
+
```

Comment:
I think I need some specification about the enum options? Also making underlay the default is not changing the current behaviour?